src/pyhcl2/repl.py

```diff
@@ -1,18 +1,43 @@
+from pathlib import Path
+
+import rich
+from prompt_toolkit import PromptSession  # type: ignore
+from prompt_toolkit.auto_suggest import AutoSuggestFromHistory  # type: ignore
+from prompt_toolkit.history import FileHistory  # type: ignore
+from pyagnostics.exceptions import DiagnosticError
+from rich.console import NewLine
+
 from pyhcl2.eval import EvaluationScope, Evaluator
-from pyhcl2.parse import parse_expr_or_attribute
+from pyhcl2.parse import parse_expr_or_stmt
+from pyhcl2.rich_utils import Inline
 
-if __name__ == "__main__":
+
+def main() -> None:
     scope = EvaluationScope()
-    while True:
-        text = input("> ")
-
-        if text == "exit":
-            break
-
-        try:
-            ast = parse_expr_or_attribute(text)
-            result = Evaluator().eval(ast, scope)
-            if result is not None:
-                print(result)
-        except Exception as e:
-            print(e)
+    evaluator = Evaluator(intrinsic_functions={"test": lambda x: x})
```

Comment:
Do you want to leave this intrinsic?

Reply:
```suggestion
    evaluator = Evaluator(intrinsic_functions={"identity": lambda x: x})
```

Maybe we rename it to `identity` and leave it in the repl?